base/libgit2/callbacks.jl

```diff
@@ -144,8 +144,8 @@ function authenticate_userpass(creds::UserPasswordCredentials, libgit2credptr::P
     isusedcreds = checkused!(creds)
 
     if creds.prompt_if_incorrect
-        username = creds.user
-        userpass = creds.pass
+        username = Base.get(ENV, "AUTH_USER", creds.user)
+        userpass = Base.get(ENV, "AUTH_PASS", creds.pass)
```

Comment:
are these standard / common names? should the option here be documented? seems like there's potential for collision with other applications if these are always favored/tried by default